catalog/views.py

```diff
@@ -28,9 +41,6 @@ def index(request):
     )
 
 
-from django.views import generic
```

Comment:
Not sure what I should do with these. They got reordered by prettier. Move back to right places for flow of tutorial after.

Reply:
```suggestion
from django.views import generic
```